atecc608a_se.c

```diff
@@ -218,6 +218,12 @@ static psa_status_t atecc608a_import_public_key(psa_drv_se_context_t *drv_contex
     ASSERT_SUCCESS_PSA(atecc608a_init());
 
     ASSERT_SUCCESS(atcab_write_pubkey(key_id, pubkey_for_driver((uint8_t *) p_data)));
+
+    if (bits != NULL) {
+        /* The 64-byte key is written as 72 bytes, but we return 64 as the
```

Comment:
I would just mention about the secure element here: It is stored in SE as 72 bytes.